include/revng/Model/NameBuilder.h

```diff
@@ -92,101 +234,214 @@ class NameBuilder {
     return *GlobalNamespace;
   }
 
-public:
   [[nodiscard]] bool isGlobalSymbol(llvm::StringRef Name) {
-    return globalNamespace().contains(Name);
+    return globalNamespace().contains(sanitize(Name));
   }
 
-public:
-  Identifier name(const model::Segment &Segment);
-  Identifier name(const model::Function &Function);
-  Identifier name(const model::DynamicFunction &Function);
+private:
+  llvm::Error addNamespaceEntryImpl(llvm::StringRef Name,
```

Comment:
I'd pusht the `addNamespaceEntry*` functions to the `.cpp` file.

Reply:
Done.